### ISSUE TITLE
 translating it into other languages

### ISSUE THREAD

**Issue by casebo** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#274]
This is a very good mod. Please make an XML file to solve the problem of translating it into other languages

**Comment by Tyler-IN** [comment]
Yeah we definitely need to start adding localization. It may come after the next release candidate, before the next full release.
